.molecule/default/setup_python.yml

```diff
@@ -0,0 +1,27 @@
+- name: Set up Python Environment
+  hosts: all
+  gather_facts: false
+
+  # run as root user
+  become: true
+  become_method: "{{ become_method }}"
+  become_user: root
+
+  tasks:
+
+  - name: pip install aiida-core requirements
+    pip:
+      chdir: "{{ aiida_core_dir }}"
+      # TODO dynamically change for python version
```

Comment:
this could be fixed once @yakutovicha has made the python version explicit in aiida-prerequisites, e.g. via an environment variable

Reply:
add it to an issue for future improvement? (bundled with https://github.com/aiidateam/aiida-core/pull/4565/files#r572159376)

Reply:
ok!

Reply:
opened #4718